linux/LinuxProcessList.c

```diff
@@ -382,11 +383,15 @@ static inline ProcessState LinuxProcessList_getProcessState(char state) {
    }
 }
 
-static bool LinuxProcessList_readStatFile(Process* process, openat_arg_t procFd, char* command, size_t commLen) {
+static bool LinuxProcessList_readStatFile(Process* process, openat_arg_t procFd, bool scanMainThread, char* command, size_t commLen) {
    LinuxProcess* lp = (LinuxProcess*) process;
 
    char buf[MAX_READ + 1];
-   ssize_t r = xReadfileat(procFd, "stat", buf, sizeof(buf));
+   char path[22] = "stat";
+   if (scanMainThread) {
+      snprintf(path, sizeof(path), "task/%"PRIi32"/stat", (int32_t)process->pid);
```

Comment:
Use `xSnprintf`, otherwise the return value is unchecked …

Reply:
Done